ui/src/main/java/com/hyperwallet/android/ui/repository/TransferMethodRepositoryImpl.java

```diff
@@ -55,6 +57,9 @@ public void createTransferMethod(final HyperwalletTransferMethod transferMethod,
             case BANK_CARD:
```

Comment:
could you please add @NonNull annotation, just keep it consistent with the other methods